### ISSUE TITLE
unable to start app

### ISSUE THREAD

**Issue by ShaneLee11** [ATF19/react-native-shop-ui#29]
did an NPM install but having problems with a bundling error. 
bundling failed: Error: Unable to resolve module `AccessibilityInfo`.

watchman was not installed. gradlew was cleaned. did some research and it could be due to RN version. no changes were done at the moment. any thoughts?

anyone encountered a similar error or have knowledge on how to solve it?

**Comment by ShaneLee11** [comment]
having issues with my gradle build. always encountering error. I have updated the sdk build version. not too sure if the issue is with react-native-maps after all the upgrades. 

anyone here with a working gradle file? could you upload the entire working folder so I could replicate it on my system? and start debugging from there? thanks

**Comment by rehankhanletsgo** [comment]
> did an NPM install but having problems with a bundling error.
> bundling failed: Error: Unable to resolve module `AccessibilityInfo`.
> 
> watchman was not installed. gradlew was cleaned. did some research and it could be due to RN version. no changes were done at the moment. any thoughts?
> 
> anyone encountered a similar error or have knowledge on how to solve it?

i also have been facing same problem
